src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.cs

```diff
@@ -33,6 +33,7 @@ internal enum TimeZoneInfoOptions
     [TypeForwardedFrom("System.Core, Version=3.5.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public sealed partial class TimeZoneInfo : IEquatable<TimeZoneInfo?>, ISerializable, IDeserializationCallback
     {
+        internal static bool Unsorted { get; } = AppContextConfigHelper.GetBooleanConfig("System.TimeZoneInfo.SystemTimeZonesUnsorted", "DOTNET_SYSTEM_TIME_ZONE_INFO_SYSTEM_TIME_ZONES_UNSORTED");
```

Comment:
>DOTNET_SYSTEM_TIME_ZONE_INFO_SYSTEM_TIME_ZONES_UNSORTED [](http://example.com/codeflow?start=137&length=55)

That is too long. maybe `DOTNET_SYSTEM_TIME_ZONES_UNSORTED` would be good enough.

Reply:
word `UNSORTED` is confusing too. maybe we can use `DOTNET_SYSTEM_TIME_ZONES_SORT_USING_ID`?

Reply:
From the note in the description, I was aiming to implement the most performant variant of `GetSystemTimeZones()`, which also avoids the cost of sorting completely.